api/nodes/views.py

```diff
@@ -231,6 +232,21 @@ def get_serializer_class(self):
         else:
             return NodeSerializer
 
+    def get_serializer_context(self):
+        context = super(NodeList, self).get_serializer_context()
+        region__id = self.request.query_params.get('region', None)
+        id = None
+        if region__id:
+            try:
+                id = Region.objects.get(_id=region__id).id
+            except Region.DoesNotExist:
+                pass
```

Comment:
It might make sense to raise a 400 here with a helpful error message like "Region {region} is invalid.". Looks like we already have a `InvalidQueryStringError` for that.

Reply:
oh okay, good to know, I was assuming we could ignore it since it was a query param, but this make sense.